core/modules/keyboard.js

```diff
@@ -138,6 +138,17 @@ function KeyboardManager(options) {
 	});
 	// Save the platform-specific name of the "meta" key
 	this.metaKeyName = $tw.platform.isMac ? "cmd-" : "win-";
+	this.shortcutKeysList = [], // Stores the shortcut-key descriptors
+	this.shortcutActionList = [], // Stores the corresponding action strings
+	this.shortcutParsedList = []; // Stores the parsed key descriptors
+	this.lookupNames = ["shortcuts"];
+	$tw.platform.isMac ? this.lookupNames.push("shortcuts-mac") : this.lookupNames.push("shortcuts-not-mac");
```

Comment:
Sorry if I missed this before, but this might be better written as:

```
this.lookupNames.push($tw.platform.isMac ? "shortcuts-mac" : "shortcuts-not-mac")
```

etc.

Reply:
@Jermolene - I've changed it